mjcf_to_sdformat/mjcf_to_sdformat/converters/sensor.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ignition.math import Quaterniond, Vector3d, Pose3d
+
+import sdformat as sdf
+import sdformat_mjcf_utils.sdf_utils as su
+
+
+def mjcf_camera_sensor_to_sdf(sensor):
+    """
+    Converts a camera sensor from MJCF to SDFormat.
+    :return: The newly created SDFormat camera sensors.
+    :rtype: [sdf.Sensor]
+    """
+    # camera following a target is not supported
+    if sensor.target is not None:
+        return None
+    sensor_sdf = sdf.Sensor()
+    if sensor.name is not None:
+        sensor_sdf.set_name(sensor.name)
+    else:
+        sensor_sdf.set_name(su.find_unique_name(sensor, "camera", "camera"))
+
+    sensor_sdf.set_type(sdf.Sensortype.CAMERA)
+    adapt_pose = Pose3d(Vector3d(), Quaterniond(-1.57, 1.57, 0))
+    pose = su.get_pose_from_mjcf(sensor) * adapt_pose
```

Comment:
Can you add a comment as to why this is needed?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/49/commits/fe5a219e2f8b07de762dd197dc2fcffce293cd2d

Reply:
I suspect Mujoco might be using a left-handed frame for cameras. To test it, I 
tried converting the following MJCF file and run it under Gazebo, but I see 
nothing in the Image Display. My hope was to compare the position of the shapes 
in the camera view between Mujoco and Gazebo. Can you give it a try?

```

<mujoco>
  <asset>
    <texture type="skybox" builtin="gradient" rgb1=".3 .5 .7" rgb2="0 0 0" width="512" height="512"/>
    <texture name="grid" type="2d" builtin="checker" width="512" height="512" rgb1=".1 .2 .3" rgb2=".2 .3 .4"/>
    <material name="grid" texture="grid" texrepeat="1 1" texuniform="true" reflectance=".2"/>
  </asset>

  <worldbody>
    <geom name="floor" size="0 0 .05" type="plane" material="grid" condim="3"/>
    <body pos="0 0 1">
      <inertial pos="0 1 0" mass="2" fullinertia="2 2 2 0 0 0"/>
      <geom name="box" type="box" pos="0.0 0 0.1" size=".3 .2 .1" />
      <geom name="sphere" pos="0.0 -5 0.1" type="sphere" size="0.5"/>
      <geom name="cylinder" pos="0.0 -5 3" type="cylinder" size="0.5 0.5"/>
    </body>

    <body name="camera_body" pos="-12 -2 3" >
      <geom name="camera_geom" type="box" size="0.1 0.1 0.1"/>
      <camera name="test_camera" pos="0.1 0 0" xyaxes="0 -1 0 0 0 1"/>
    </body>
  </worldbody>
</mujoco>
```